viral/sequence_utils.py

```
@@ -634,6 +675,187 @@ def calculate_radon_replay(
+    # is in centimetres (originally in metres)
+    # TODO: will this have to be changed for our different experimental setup?
+    # -> test for intercepts betweeen -1500 cm and 2100 cm in 1 cm increments
+    # TODO: even in their MATLAB code I can't see them use this -> weird??
+    candidate_cs = np.arange(-1500, 2101, 1)
+
+    # % minLgth         =size(pMat,2)/3;
+    # TODO: think about this, could be an absolute value or a percentage, is 1/3 of the entire temporal bins at the moment
+    min_path_length = n_temporal_bins / 3
+
+    results = list()
+    for i, V in enumerate(candidate_Vs):
```

Ruff (0.15.6)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)